packs/data/feats.db/lavasoul.json

```diff
@@ -52,7 +54,8 @@
                     "criticalSuccess"
                 ],
                 "selector": "{item|_id}-damage",
-                "text": "Instead of the normal critical specialization effect, the target takes [[/r 1d6[persistent,fire]]] damage fire damage; you gain a bonus on this persistent damage equal to your item bonus to attack rolls on unarmed attacks."
+                "text": "PF2E.SpecificRule.Ifrit.Lavasoul.MagmaSpikeNote",
```

Comment:
We can make this an alternate crit spec instead of a note, see the RE on Spike Launcher.